frontend/src/hocs/withScreencast/withScreencast.tsx

```diff
@@ -77,51 +69,49 @@ function withScreencast(
       currentState: "OFF" as Steps,
     }
 
-    toggleRecordAudio = (): void => {
+    private toggleRecordAudio = (): void => {
       const { recordAudio } = this.state
 
       this.setState({
         recordAudio: !recordAudio,
       })
     }
 
-    showDialog = (fileName: string): void => {
+    private showDialog = (fileName: string): void => {
       const { currentState } = this.state
 
-      if (!this.checkSupportedBrowser()) {
-        this.setState({
-          currentState: "UNSUPPORTED",
-        })
+      if (!ScreenCastRecorder.isSupportedBrowser()) {
+        this.setState({ currentState: "UNSUPPORTED" })
```

Comment:
Nit: I think enums are more suited for these kinds of things than union types. Thoughts?

Reply:
I'm agnostic! I don't know of a good reason to support one over the other - though there might be one. What is your preference based on?